README.md

```diff
@@ -1,2 +1,246 @@
+NUMBER_OF_NODES: Total number of nodes running not only Marklogic database, but also nodes running other applications.
+
+# Install Marklogic Helm Chart
+
+## Add Marklogic Repo
+
+If you haven’t already, add the Marklogic official repo to Helm.
+
+```
+helm repo add marklogic https://github.com/marklogic/marklogic-kubernetes
+```
+
+The output will look like this:
+
+```
+"marklogic" has been added to your repositories
+```
+
+Use this command to verify that the repo has been added to Helm:
+
+```
+helm repo list
+```
+
+You should see an entry like this:
+
+`marklogic           https://github.com/marklogic/marklogic-kubernetes`
+
+Use this command to ensure the Helm repo is up to date:
+
+```
+helm repo update
+```
+
+## Installing the Chart
+
+Use this command to install marklogic chart to current namespace with default settings:
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1
+```
+
+After you install, the output would look like this:
+```
+NAME: my-release
+LAST DEPLOYED: 
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+```
+
+**Note:** --version=1.0.0-ea1 must be provided. You can choose a distinctive release name to replace "my-release"
+
+It's strongly recommend to deploy marklogic chart in exclusive namespace, use --create-namespace flag if the namespace have not be created:
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1 --namespace=marklogic --create-namespace
+```
+
+Use this command to verify the deployment:
+
+```
+helm list --all-namespaces
+```
+
+You should see an entry named "my-release" (or the release name you chose) with a status of "deployed".
+
+## Configuration Options for Installation
+
+This section describes the configuration options you can use with Helm. 
+
+### --values
+
+The --values flag points to a YAML file. The values in the file override the default Helm values.
+
+Use this command to view the default configurable values:
+
+```
+helm show values marklogic/marklogic --version=1.0.0-ea1
+```
+
+To configure a different value for your installation, first you need to create a values.yaml file.
+
+For example, if you want to set the credential for Docker Hub, configure the values.yaml file like this:
+
+```
+imagePullSecret: 
+  registry: "https://index.docker.io/v1/"
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Use the following command to install MarkLogic with the values.yaml you just created.
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1 --values values.yaml
+```
+
+### --set
+
+Use the --set flag to make one or more configuration changes directly:
+
+```
+helm install my-release marklogic/marklogic --version=1.0.0-ea1 \
+--set imagePullSecret.registry="https://index.docker.io/v1/" \
+--set imagePullSecret.username=YOUR_USERNAME \
+--set imagePullSecret.password=YOUR_PASSWORD
+```
+
+We recommend that you use the  values.yaml for configuring your installation.
```

Comment:
We recommend that you use the `values.yaml` file for configuring your installation.